### ISSUE TITLE
[Main] NetworkManager Settings is broken: /usr/lib/libnm.so.0: version libnm_1_40_0 not found

### ISSUE THREAD

**Issue by popsUlfr** [ValveSoftware/SteamOS#1108]
#### Your system information

* Steam client version: **Stable/1689034492**
* SteamOS version: **Main/20230727.1000**
* Opted into Steam client beta?: **No**
* Opted into SteamOS beta?: **Main** 
* Have you checked for updates in Settings > System?: **Yes**

#### Please describe your issue in as much detail as possible:

Going to the network configurations in desktop mode results in a broken settings panel:

```
Could not load plugin from /usr/lib/qt/plugins/plasma/kcms/systemsettings_qwidgets/kcm_networkmanagement.so: Cannot load library /usr/lib/qt/plugins/plasma/kcms/systemsettings_qwidgets/kcm_networkmanagement.so: (/usr/lib/libnm.so.0: version `libnm_1_40_0' not found (required by /usr/lib/qt/plugins/plasma/kcms/systemsettings_qwidgets/kcm_networkmanagement.so))
```

![Screenshot_20230728_112216](https://github.com/ValveSoftware/SteamOS/assets/1030423/8a69620d-a811-4341-8147-19f3b3eaf50a)

#### Steps for reproducing this issue:

1. Click the network icon in the system tray
2. Click on `Configure Network Connections...`
3. It brings you to a broken System Settings Panel

**Comment by popsUlfr** [comment]
I'm fixing this manually by doing:

```sh
sudo pacman --cachedir /tmp -Sw extra-main/libnm
sudo steamos-readonly disable
for f in /usr/lib/libnm.so* ; do sudo mv -v "$f" "${f}.orig" ; done
sudo tar -xvpf /tmp/libnm-*.pkg.tar.zst --wildcards -C / 'usr/lib/libnm.so*'
sudo steamos-readonly enable
```

There's collision going on between the `jupiter-main` and `extra-main` repos.

**Comment by bertogg** [comment]
FYI this is being taken care of.

**Comment by w1lddog2** [comment]
> ```shell
> sudo tar -xvpf /tmp/libnm-*.pkg.tar.zst --wildcards -C / 'usr/lib/libnm.so*'
> ```

That did it.

**Comment by popsUlfr** [comment]
Appears to be fixed on my end on  **20230907.1000**.

Thank you :)

**Comment by bertogg** [comment]
> Appears to be fixed on my end on 20230907.1000.

That's correct, thanks for confirming it.

**Comment by bertogg** [comment]
Also fixed on SteamOS 3.5 preview (build: 20230915.100)

**Comment by kisak-valve** [comment]
Closing as fixed.
